tools/releases/dockerfiles/Dockerfile.kuma-dp

```diff
@@ -1,10 +1,11 @@
-FROM frolvlad/alpine-glibc:alpine-3.15_glibc-2.33
+FROM gcr.io/distroless/base-debian11:debug
```

Comment:
> can you tell us a little bit more about this image?

yes, I looked up how Envoy did this, they decided to deprecate alpine images and use `distroless` instead, here is comment that explains things little bit https://github.com/envoyproxy/envoy/issues/16170#issuecomment-826957603

> why do we need to switch to this image?

we have to switch the image, because there is no arm64 support for `frolvlad/alpine-glibc:alpine-3.15_glibc-2.33`

> what is debug tag? it does not sound stable

Regual distroless image is so minimalistic it doesn't even have `shell` in it. I don't think it worth it taking into account the difference in sizes:
```
gcr.io/distroless/base-debian11        nonroot                                d81155b67689   52 years ago     17.4MB
gcr.io/distroless/base-debian11        debug-nonroot                          5aa6121e6dc7   52 years ago     18.6MB
```
Debug version adds shell, more info here https://github.com/GoogleContainerTools/distroless#debug-images

> is there an alpine arm64 image? why other kuma images could preserve existing image?

Yes, but we need glibc for Envoy, that's why it only for `kuma-dp`